tests/integration/test_wazuh_db/data/global_messages.yaml

```diff
@@ -99,7 +99,7 @@
     stage: "global update-keepalive success"
   -
     input: 'global get-agent-info 1'
-    output: 'ok [{"id":1,"name":"TestName2","ip":"0.0.0.1","register_ip":"0.0.0.1","internal_key":"TopSecret","os_name":"TestOsName","os_version":"TestOsVersion","os_major":"TestOsMajor","os_minor":"TestOsMinor","os_codename":"TestOsCodeName","os_build":"TestOsBuild","os_platform":"TestOsPlatfor","os_uname":"TestOsUname","os_arch":"TestOsArch","version":"TestVersion","config_sum":"TestConfigSum","merged_sum":"TestMergedSum","manager_host":"TestManagerHost","node_name":"TestNodeName","date_add":1599223378,"last_keepalive":*,"status":"updated","fim_offset":100,"reg_offset":100,"group":"TestGroup2","sync_status":"syncreq"}]'
```

Comment:
Considering that just before this stage there was an `update-keepalive`. Shouldn't be better to have an `active` state? I know that up to this point the `update-keepalive` command was not modified to also set the `connection_status` to `active`, but this will happen in the near future so, it would be good if we update the test cases thinking on that.

Reply:
Please, don't take into account this comment because you don't have a way/mechanism to have the status as `active` in this moment.

Reply:
We can do that soon.
Thank you!